.github/workflows/gh-pages.yml

```diff
@@ -54,9 +54,8 @@ jobs:
       - name: Build the doc
         run: |
           cargo xtask doc
+          mv target/doc target/guide/doc
```

Comment:
Don't we need to `mkdir target/guide` for this to work?

Reply:
`target/guide` is created in the previous "Build the guide" step - indeed this has passed in the CI `guide-build` job :)